CS483/CS483/Kartaclysm/Components/ComponentKartController.cpp

```diff
@@ -469,11 +486,11 @@ namespace Kartaclysm
 		// swerve temporarily disabled until the camera transform heirarchy is fixed
 		if (m_bWheelie)
 		{
-			m_pGameObject->GetTransform().SetRotation(glm::quat(glm::vec3(m_fWheelieRotation, m_fDirection + GetRotationMod(), 0.0f)));
+			m_pGameObject->GetTransform().SetRotation(glm::quat(glm::vec3(m_fWheelieRotation + m_fOffroadRumble, m_fDirection + GetRotationMod(), 0.0f)));
```

Comment:
BTW, rumble looks hilarious with the wheelie activated. Needs to be fixed at some point (read: backlog)